### ISSUE TITLE
Favicon not working

### ISSUE THREAD

**Issue by HettieM** [fac19/week10-hettiehannah#21]

**Comment by akomiqaia** [comment]
the file you have seems not to be good. i used other ico file and it worked. also in inex HTML i changed the src to this and it worked:
![Screenshot from 2020-05-08 10-17-32](https://user-images.githubusercontent.com/31027725/81391540-289dea80-9115-11ea-9afb-6df7b67d60e7.png)

I am not sure why but i think this is connected to whats going on in manifest.js in public folder.

**Comment by freemvmt** [comment]
I agree, the problem is with the favicon file itself! And all the image files in fact - they seem to be corrupted somehow. I can't view them in vscode or my file browser.

fyi @akomiqaia I think the manifest.js file only comes into play if you're making a [PWA](https://web.dev/progressive-web-apps/)
